app/src/main/java/org/simple/clinic/drugs/selectionv2/PrescribedDrugsScreenControllerV2.kt

```diff
@@ -60,25 +60,22 @@ class PrescribedDrugsScreenControllerV2 @Inject constructor(
         .map { (protocolDrugs, prescribedDrugs) ->
 
           val prescribedProtocolDrugs = prescribedDrugs.filter { it.isProtocolDrug }
-          // Select protocol drugs if prescriptions exist for them.
+          val isLast = prescribedDrugs.any { it.isProtocolDrug.not() }
```

Comment:
Rename `isLast` -> `isAtLeastOneNonProtocolDrugPrescribed`?

Reply:
You mean `isAtleastOneCustomDrugPrescribed`